### PR TITLE
[docs] Update reference for various cleos commands - 2.1

### PR DESCRIPTION
**Pull request by lparisc**
## Change Description

Combine PRs #10200 #10206 #10209 #10212 #10215 #10217 to 2.1
Minor edits to the reference of various cleos commands

## Change Type
**Select *ONE*:**
- [x] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: docs/02_cleos/03_command-reference/create/account.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
+## Command 
+
+```ssh
+cleos create account [OPTIONS] creator name OwnerKey [ActiveKey
```
Comment (halsaphi) [inline docs/02_cleos/03_command-reference/create/account.md]: missing last ]

File: docs/02_cleos/03_command-reference/create/account.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+[[info | Note]]
+| The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Create a new account on the blockchain (assumes system contract does not restrict RAM usage)
+Use this command to create a new blockchain account on EOSIO. 
 
-## Usage
+[[info | Note]]
+| This command assumes system contract does not restrict RAM usage.
```
Comment (halsaphi) [inline docs/02_cleos/03_command-reference/create/account.md:18]: I think this needs an explanation -> or a link to something explaining ram and accounts, and an accounts requirement for ram and the cost etc

File: docs/02_cleos/03_command-reference/create/account.md [hunk 3]
```diff
@@ -29,16 +50,14 @@ Options:
...
+For prerequisites to successfully run this command, see the **Before you Begin** section of the [_How to Create a Blockchain Account_](../02_how-to-guides/how-to-create-an-account.md) guide.
 
-```sh
-cleos create account inita tester EOS4toFS3YXEQCkuuw1aqDLrtHim86Gz9u3hBdcBw5KNPZcursVHq EOS7d9A3uLe6As66jzN8j44TXJUqJSK3bFjjEEqR4oTvNAB3iM9SA
-```
+## Example Command
 
-## Output
+```console
+cleos create account inita tester EOS4toFS3YXEQCkuuw1aqDLrtHim86Gz9u3hBdcBw5KNPZcursVHq EOS7d9A3uLe6As66jzN8j44TXJUqJSK3bFjjEEqR4oTvNAB3iM9SA
```
Comment (halsaphi) [inline docs/02_cleos/03_command-reference/create/account.md:53]: does this need the where bit to explain the parameters - help explain the 2 public keys

File: docs/02_cleos/03_command-reference/get/account.md [hunk 4]
```diff
@@ -58,7 +73,9 @@ cpu bandwidth: (averaged over 3 days)
 
 producers:     <not voted>
 ```
-2. Get JSON data for user **eosio**
+
+
+**Example 2: Retrieve formatted JSON data for `eosio` account**
```
Comment (halsaphi) [inline docs/02_cleos/03_command-reference/get/account.md]: do you need the word formatted here?

File: docs/02_cleos/03_command-reference/wallet/import.md [hunk 5]
```diff
@@ -1,55 +1,36 @@
...
-**Where**
-* [OPTIONS] = See Options in Command Usage section below. 
+**Where**:
 
-**Note**: The arguments and options enclosed in square brackets are optional.
+* [`OPTIONS`] = See **Options** in **Command Usage** section below
 
 ## Description
-Imports private key into wallet. This command will launch `keosd` if it is not already running. 
+Use this command to imports private key into wallet.
```
Comment (halsaphi) [inline docs/02_cleos/03_command-reference/wallet/import.md:11]: imports => import

Comment (halsaphi) [reply docs/02_cleos/03_command-reference/wallet/import.md:11]: Use this command to imports private key into wallet.  to
=>
Use this command to import a  private key into a (local maybe?) wallet.

**Review by halsaphi — APPROVED** [review]

**Review by bobgt — APPROVED** [review]

**Pull request merged**